DEVELOPERS.md

```diff
@@ -192,7 +198,7 @@ r_core_wrap.cxx:32103:61: error: assigning to 'RDebugReasonType' from incompatib
 
 * See doc/vim for vimrc
 
-* See doc/clang-format for work-in-progress support for automated indentation
+* See .clang-format for work-in-progress support for automated indentation
```

Comment:
then we are not distributing it at install time. is this expected?

Reply:
yep